internal/docs/release.md

```diff
@@ -4,6 +4,19 @@ We aim to do a release after each Sprint (roughly, every 3 weeks).
 
 To do a release:
 
+-   Do the prerelease checks until everything is passing.
+    -   Create a new branch (`git branch -B prerelease`). Running the script
+        will modify hundreds of golden files, so you want to do it in a branch.
+        You can delete the branch and abandon all the changes when you are done.
+    -   Run `./internal/testing/prerelease.sh init` until it finishes with
+        `SUCCESS`. Note that there are some known failures that are logged but
+        don't prevent overall `SUCCESS`.
+    -   Run `./internal/testing/prerelease.sh run` Again, there are some known
+        failures. You can re-run any unexpected failures independently to
+        investigate; see the `prerelease.sh` script to see the exact command
+        line to use.
+    -   Run `./internal/testing/prerelease.sh cleanup` when you are done.
+    -   Commit the changes to the branch, and then delete the branch.
```

Comment:
Why would you commit changes to a branch and then delete it?

Reply:
There may be a better way, but that's how I do it. You can't delete a branch while you're in it, and if you move out of the branch without committing the changes, they come along with you and aren't dropped when you delete the branch.